VideoDuplicateFinder.Windows/ViewModels/DuplicateViewModel.xaml

```diff
@@ -37,7 +37,11 @@
 
         <Image
             Grid.Column="1"
-            Margin="10,0,0,0"
+            Margin="10,-10,0,-10"
```

Comment:
When changing the image width here but also in scan engine the Linux version needs to be updated as well